### PR TITLE
Release note hotfix

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: enterprise/v0.26/10_resources/01_release-notes.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Astronomer v0.26 is the latest Stable version of Astronomer Enterprise, while v0
 
 We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
-## 0.26.6
+## 0.26.5
 
-Release date: December 11, 2021
+Release date: December 10, 2021
```
Comment (vishwas-astro) [inline enterprise/v0.26/10_resources/01_release-notes.md:17]: Release date should still be Dec 11

**Review by vishwas-astro — APPROVED** [review]

**Comment by paolaperaza** [comment]
image

@jwitz Know it's a Saturday but can confirm that Dec 11 is still the correct release date!

Ship it once you make that change! 🚢

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ We're committed to testing all Astronomer Enterprise versions for scale, reliabi
 
 ## 0.25.12
 
-Release date: December 11, 2021
+Release date: December 10, 2021
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:17]:
```suggestion
Release date: December 11, 2021
```

**Review by paolaperaza — APPROVED** [review]

File: enterprise/v0.26/10_resources/01_release-notes.md (hunk 1 above)
Comment (jwitz) [inline enterprise/v0.26/10_resources/01_release-notes.md:17]:
```suggestion
Release date: December 11, 2021
```

**Pull request merged**